main.go

```diff
@@ -5,21 +5,35 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"time"
 
 	"github.com/achiku/mux"
+	"cloud.google.com/go/profiler"
 )
 
+const (
+	serviceName = "ogp-app"
+	serviceVersion = "1.0.0" // TODO: replace this
+	maxRetry = 3
+)
+
+func init() {
```

Comment:
Do we need this?

Reply:
Sorry I forgot to remove that part. The intention here was to check necessary directories before `main()` and avoid the error from `os.Open()`. I'll raise another issue just to tackle that.